src/lib/queue_worker.js

```diff
@@ -97,7 +97,7 @@ QueueWorker.prototype._getLogEntry = function(message) {
  *   reference to the Firebase location of the task that's timed out.
```

Comment:
Maybe add a `@param` string here explaining `immediate` as a boolean for whether to check the taskTimeout or not.